### PR TITLE
XW-1242 add check for category alert

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/mercury/old/MercuryAlertComponentObject.java [hunk 1]
```diff
@@ -1,46 +1,50 @@
...
+    AlertMessage(String message) {
+      this.message = message;
+    }
+
+    public String getMessage() {
+      return message;
+    }
+  }
+
+  private Wait wait;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/mercury/old/MercuryAlertComponentObject.java]: MINOR Move this variable to comply with Java Code Conventions. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1213)

File: src/test/java/com/wikia/webdriver/elements/mercury/old/MercuryAlertComponentObject.java [hunk 2]
```diff
@@ -1,46 +1,50 @@
...
+
+  public MercuryAlertComponentObject(WebDriver driver, AlertMessage message) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/mercury/old/MercuryAlertComponentObject.java]: MINOR Move this constructor to comply with Java Code Conventions. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1213)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Comment by hakubo** [comment]
LGTM

**Pull request merged**